variants/tbeam/platformio.ini

```diff
@@ -6,4 +6,5 @@ lib_deps =
   ${esp32_base.lib_deps}
 build_flags = 
   ${esp32_base.build_flags} -D TBEAM_V10  -I variants/tbeam
+  ;-DGPS_POWER_TOGGLE ;uncomment this line to allow a double press on the user button to turn off gps entirely.
```

Comment:
I think we can uncomment this for the commit. It's a good thing to have and its tbeam only.